target/linux/lantiq/xrx200/config-default

```diff
@@ -49,3 +49,5 @@ CONFIG_USB_SUPPORT=y
 CONFIG_XRX200_PHY_FW=y
 CONFIG_ZLIB_DEFLATE=y
 CONFIG_ZLIB_INFLATE=y
+CONFIG_HWMON=y
+CONFIG_SENSORS_LTQ_CPUTEMP=y
```

Comment:
you will need to add 
# CONFIG_SENSORS_LTQ_CPUTEMP is not set to target/linux/lantiq/config-4.4 otherwise this will throw an error on all !xrx200 targets